examples/index.json

```diff
@@ -805,11 +805,11 @@
         "category": "Working over the network",
         "samples": [{
                 "name": "HTTP 1.1 to 2.0 Protocol Switch",
-                "url": "http-1.1-to-2.0-protocol-switch"
+                "url": "http-1_1-to-2_0-protocol-switch"
             },
             {
                 "name": "HTTP 2.0 Server Push",
-                "url": "http-2.0-server-push"
+                "url": "http-2_0-server-push"
```

Comment:
here underscore ("_") should be replaced by dash ("-")

Reply:
Done